cli/azd/cmd/testdata/TestUsage-azd-publish.snap

```diff
@@ -0,0 +1,37 @@
+
+Publish a service to a container registry.
+
+  • Supports Container App services only.
+  • Target registry set by AZURE_CONTAINER_REGISTRY_ENDPOINT environment variable or docker.registry in azure.yaml.
```

Comment:
Is `--to [registry]/image` a valid use case with the latest changes?

Reply:
Yes, updated the bullet point - thanks.